content/applications/hr/appraisals.rst

```diff
@@ -0,0 +1,507 @@
+:guilabel:`Final Rating` field also appears once the appraisal request is confirmed.
+
+If there are any existing appraisals for the employee, an :guilabel:`Appraisal` smart button appears
+at the top, listing the total number of appraisals there are for the employee.
+
+Ask for feedback
+~~~~~~~~~~~~~~~~
+
+As part of the appraisal process, the manager can request feedback on an employee from anyone in the
+company. Feedback is usually requested from coworkers and other people who interact with or work
+with the employee. This is to get a more well-rounded view of the employee and aid in the manager's
+assessment.
+
+To request feedback, click the :guilabel:`Ask Feedback` button at the top of the form. An email form
+appears using the :guilabel:`Appraisal: Ask Feedback` email template, which sends the :guilabel:`360
+Feedback` survey. Enter the employees being asked to complete the survey in the
+:guilabel:`Recipients` field. Multiple employees may be selected. The email template has dynamic
+placeholders to personalize the message. Add any additional text to the email if desired. If any
+attachments are needed, click the :guilabel:`Attachments` button and a file explorer window appears.
+Navigate to the file(s), select them, then click :guilabel:`Open`. When the email is ready to send,
+click :guilabel:`Send.`
+
+.. image:: appraisals/ask-feedback.png
+   :align: center
+   :alt: The email pop-up when requesting feedback from other employees.
+
+Appraisal form
+~~~~~~~~~~~~~~
+
+Once an appraisal is confirmed, the next steps are for the employee to fill out the self-assessment,
+after which the manager completes their assessment.
+
+.. _appraisals/employee-feedback:
+
+Employee's feedback
+*******************
+
+Navigate to the main :menuselection:`Appraisals application` dashboard. The only entries visible on
+the dashboard are appraisals for the employee themselves, and anyone they manage and have to provide
+manager feedback for.
+
+Click on the appraisal to open the appraisal form. Enter responses in the :guilabel:`Employee's
+Feedback` section. When completed, click the :guilabel:`Not Visible to Manager` toggle (the default
+setting once an appraisal is confirmed). The toggle changes to :guilabel:`Visible to Manager`.
+
+.. _appraisals/manager-feedback:
+
+Manager's feedback
+******************
+
+After the employee has completed the :guilabel:`Employee's Feedback` section, it is time for the
+manager to fill out the :guilabel:`Manager's Feedback` section. The manager enters the responses in
+the fields in the :ref:`same manner as the employee <appraisals/employee-feedback>`. When the
+feedback section is completed, click the :guilabel:`Not Visible to Employee` toggle (the default
+setting once an appraisal is confirmed). The toggle changes to :guilabel:`Visible to Employee`.
+
+.. image:: appraisals/feedback.png
+   :align: center
+   :alt: The feedback section for both employees and managers. The toggle buttons are highlighted.
```

Comment:
True- I had a hard time figuring out what to do with this since there was SO much text, and I wanted to show how it looked (side by side). This suggestion is much better- and people following along should understand.